src/commands/mod.rs

```diff
@@ -59,6 +62,12 @@ pub enum Command {
     /// Signs a ManageNeuron message to submit a UpgradeSnsControlledCanister
     /// proposal.
     MakeUpgradeCanisterProposal(make_upgrade_canister_proposal::MakeUpgradeCanisterProposalOpts),
+    /// Prints a summary with general info about the sns.
```

Comment:
I think each of these command descriptors can be improved. I encourage you to really think about what description you'd want to see to understand what this command would do if you were encountering this tool for the first time. These comments are what clap uses to print info about what the commands do when you run `sns-quill --help`

Reply:
Done